test/unit/modules/observer/observer.spec.js

```diff
@@ -268,6 +268,17 @@ describe('Observer', () => {
     // should ignore deleting non-existing key
     delProp(obj1, 'a')
     expect(dep1.notify.calls.count()).toBe(3)
+    // should ignore notifications if { notify: false } is set
+    setProp(obj1, 'd', 1, {}) // should notify
+    setProp(obj1, 'e', 1, { foo: 'bar' }) // should notify
+    setProp(obj1, 'f', 1, { notify: true }) // should notify
+    setProp(obj1, 'g', 1, { notify: false }) // should NOT notify
```

Comment:
Acknowledged. I changed the PR to not use the `{notify:false}` API anymore, but I still have it saved in a branch if we want to restore it.